docs/external plugins/Verifier/sbom.md

```diff
@@ -0,0 +1,294 @@
+# SBOM Validation
+
+This document outlines how Ratify can be used to verify sbom (Software bill of material). The `sbom` verifier is added as a plugin to the Ratify verification framework.  Currently the sbom verifier is in 2.0.0-alpha release, and it supports the following sbom validation:
+- sbom attached to the subject image as a referrer artifact
```

Comment:
general comment: should we capitalize all references to `sbom` since it's an acronym?

Reply:
fixed. thanks!